index.html

```diff
@@ -17,14 +24,15 @@
         </div>
         <div class="form-field">
           <label for="level" class="form-label">Level*</label>
-          <select name="level" class="form-input">
+          <select id="level" name="level" class="form-input">
```

Comment:
Adding id attributes to form elements enhances accessibility and makes it easier to manipulate these elements with JavaScript.